lib/shopify-cli/commands/config.rb

```diff
@@ -39,6 +44,33 @@ def call(args, _name)
           end
         end
       end
+
+      class Analytics < ShopifyCli::SubCommand
```

Comment:
This works overall, but there is a fair amount of duplication involved.

It seems to me like `analytics` calls are essentially `feature` calls where the `feature_name` is `analytics`. I would consider changing this so that analytics just proxies the call to the base code with the appropriate feature name.

I'd even suggest inheriting from the `Feature` class so we get the flag parsing as a bonus.

Reply:
Tim and I considered doing this, but thought that anonymous data reporting isn't _really_ a feature? This would be a really easy fix though, so let me know if it's more appropriate to put it under the Feature class!

Reply:
I think this boils down to personal preference, and personally I would have an Analytics class that calls through to Feature (data reporting is a feature of the CLI IMHO). I believe that would separate them enough that we could easily add custom behaviour but still avoid duplicating everything but the name.

I'm not dead set on doing it that way though! What we have now works as well.

Reply:
@tanema do you object to making `Analytics.call` proxy into `Feature.call` to DRY this up?

Reply:
Please disregard all of the above, taking a deeper look into this showed me that the code to enable / disable the features is actually different, so the duplication makes sense.

Sorry for the false alarm!